cmd/tusd/cli/log.go

```diff
@@ -7,9 +7,12 @@ import (
 	"github.com/tus/tusd"
 )
 
-var stdout = log.New(os.Stdout, "[tusd] ", 0)
 var stderr = log.New(os.Stderr, "[tusd] ", 0)
 
-func logEv(eventName string, details ...string) {
-	tusd.LogEvent(stderr, eventName, details...)
```

Comment:
As far as I can tell, logEv is only used for events which should be written to stdout. So I think it should be enough to change this line to `tusd.LogEvent(stderr, eventName, details...)` without the rest of your changes. Could it be that I am missing something?

Reply:
Nope, it can definitely be like that, I just wasn't sure if there was some coding standard to use logging as a global variable. I'll change it.

Reply:
Actually, looking at this again @Acconut, https://github.com/tus/tusd/pull/171/files#diff-87d4bf0db9b6383dae738a3c1168365eR99 should be logging to stderr, since it's actually reporting a hook error.

Reply:
Ok, I am also fine with that :+1: